jellyfin-api/src/commonMain/kotlin/org/jellyfin/sdk/api/sockets/listenerRegistration.kt

```diff
@@ -0,0 +1,117 @@
+package org.jellyfin.sdk.api.sockets
```

Comment:
Even if it's all extension functions, shouldn't the filename still be upper-case?